site/content/3.10/data-science/adapters/arangodb-rdf-adapter.md

```diff
@@ -0,0 +1,170 @@
+            "Actor": {"name"}
+        },
+        "edgeCollections": {
+            "eventActor": {}
+        },
+    },
+)
+```
+
+## Terminology
+
+### Literals
+
+In RDF, even literal values are referenced by edges. Literals cannot have
+outgoing edges (i.e., cannot be the subject of a statement). RDF uses the XSD
+type system for literals, so the string "Fred" is represented as `"Fred"^^xsd:String` 
+or fully expanded as `"Fred" ^^http://…"`. Literals can also contain language 
+and locale tags, for example, `"cat@en" ^^xsd:String` and `"chat@fr"^^xsd:String`. 
+These language tags can be useful and would ideally be preserved. 
+
+Literals could be added as a property instead of creating a separate vertex; this 
+takes better advantage of using a property graph. If you are coming from a triple 
+store or downloading your data using a [SPARQL](https://www.w3.org/TR/rdf-sparql-query/) query you could handle these properties when 
+exporting. 
+
+### Uniform Resource Identifiers (URIs)
+
+#### Prefixes
+
+In RDF, it is common to use [namespace prefixes](https://www.w3.org/TR/rdf-concepts/#section-URIspaces) with references for ease of parsing. This 
+can be easily handled with a property graph in a few ways. The easiest approach 
+is to add the statement prefixes to the document. This keeps the prefixes close 
+to the data but results in a lot of duplicated fields. Another approach would be 
+to append the prefix and form the full URI as a property.
+
+#### Identifiers
+
+URIs (e.g `http://dbpedia.org/resource/`) are used as universal identifiers in 
+RDF but contain contain special characters, namely `:` and `/`, which make them not 
+suitable for use as an ArangoDB `_key` attribute. Consequently, a hashing algorithm
+is used within `ArangoRDF` to store the URI as an ArangoDB Vertex.
+
+### Blank Nodes
+
+Blank nodes are identifiers that have local scope and cannot (must not) be
+referenced externally. Blank nodes are usually used in structures like lists and 
+other situations where it is inconvenient to create URIs. They will cause problems
+when reconciling differences between graphs. Hashing these values as well is a way 
+to work around them but as they are considered temporary identifiers in the RDF 
+world they could pose consistency issues in your RDF graph.
+
+### Serializations
+
+There are numerous RDF serializations, including XML and JSON based
+serializations and gzipped serializations. Third party libraries will likely handle 
+all of the serializations but it is a step that may effect how data is imported. 
+
+### Ontology, Taxonomy, Class Inheritance, and RDFS
+
+The final consideration is something that for many is the core of RDF and 
+semantic data: [Ontologies](https://www.w3.org/standards/semanticweb/ontology).
+Not just ontologies but also class inheritance, and schema validation. One method 
+would be add the ontology in a similar way to what has been suggested for the 
```

Comment:
should it be 'to add'?

Reply:
thanks: https://github.com/arangodb/docs-hugo/pull/444